source/boards/boards-settings.rst

```diff
@@ -0,0 +1,17 @@
+Settings
+========
+
+Set language
+------------
+
+To set your language on boards, select the gear icon next to your profile avatar, then go to **Set language** to apply your language settings. Please note: language settings in Boards are independent from language settings in Channels.
```

Comment:
```suggestion
To set your preferred language for Mattermost Boards, select the gear icon next to your profile avatar, then go to **Set language** to apply your language preference. Please note: language settings in Boards are independent from language settings in Channels.
```

Reply:
This is a good place to include the Gear icon. See the [Customize your Mattermost experience](https://docs.mattermost.com/welcome/get-started-mattermost-channels.html#customize-your-mattermost-experience) docs & source page for a working example.

Also consider formatting the "please note" text as a note directive.